ssh/tailssh/incubator.go

```diff
@@ -54,11 +54,15 @@ var maybeStartLoginSession = func(logf logger.Logf, uid uint32, localUser, remot
 
 // newIncubatorCommand returns a new exec.Cmd configured with
 // `tailscaled be-child ssh` as the entrypoint.
-// If tailscaled is empty, the desired cmd is executed directly.
-func newIncubatorCommand(ctx context.Context, ci *sshConnInfo, lu *user.User, tailscaled, name string, args []string) *exec.Cmd {
-	if tailscaled == "" {
+//
+// If ss.srv.tailscaledPath is empty, this method is equivalent to
```

Comment:
I didn't want to pass the server down to session, well especially not have it access private fields. You can either make sshSession also have `tailscaledPath`, or create a processLauncher

Reply:
processLaucher will actually help us test this code better in general.

Reply:
I definitely _do_ want an up pointer back up to the server from sshSession. That's a very common pattern.  (e..g. peerAPIHandler up to peerAPIServer up to LocalBackend)

I don't want to copy a rando `tailscaledPath` into sshSession when it'll never vary per session or really have anything to do with a session.